db.py

```diff
@@ -1,46 +1,70 @@
 import json
+import os
 import sqlite3
 from collections import Counter
 
-# Defining the path of the SQLite database file
-DATABASE = 'data.db'
-DATABASE_ESP = 'esp.db'
-DATABASE_SETTING = 'settings.db'
-DATABASE_TAG = 'tags.db'
-
-
-# Function to connect to the database
-def get_db():
-    conn = sqlite3.connect(DATABASE)
-    conn.row_factory = sqlite3.Row
-
-    # Create the items table if it does not exist
-    conn.execute('''
-        CREATE TABLE IF NOT EXISTS items (
-            id INTEGER PRIMARY KEY AUTOINCREMENT,
-            name TEXT NOT NULL,
-            link TEXT NOT NULL,
-            image TEXT,
-            position TEXT,
-            quantity INTEGER,
-            ip TEXT,
-            tags TEXT 
-        )
-    ''')
-    conn.commit()
-    return conn
+# Define the path for the combined database
+COMBINED_DATABASE = 'mimosa_V4_database.db'
```

Comment:
```suggestion
os.makedirs("./data", exist_ok=True)
COMBINED_DATABASE = './data/mimosa_V4_database.db'
```
while this is there, you could place the database in a subdirectory so that it is easier to store when using docker, and is usually a good idea anyways.

Reply:
@FireMarshmellow Actually, you might want to do this since it's your commit